db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1879,6 +1879,20 @@ Status DBImpl::RunManualCompaction(
       assert(!exclusive || !manual_conflict);
       // Running either this or some other manual compaction
       bg_cv_.Wait();
+      if (manual_compaction_paused_ > 0 && !manual.done &&
+          !manual.in_progress) {
```

Comment:
Is my understanding right that for `in_progress == true` the cleanup / failure can be expected to arise in the background?

Reply:
that's right.